lib/phoenix_html/form.ex

```diff
@@ -631,11 +631,28 @@ defmodule Phoenix.HTML.Form do
   """
   @spec hidden_inputs_for(t) :: list(Phoenix.HTML.safe())
   def hidden_inputs_for(form) do
-    for {k, v} <- form.hidden do
-      hidden_input(form, k, value: v)
+    Enum.flat_map(form.hidden, fn {k, v} ->
+      hidden_inputs_for(form, k, v)
+    end)
+  end
+
+  defp hidden_inputs_for(form, k, values) when is_list(values) do
+    id = input_id(form, k)
+    name = input_name(form, k)
+
+    for {v, index} <- Enum.with_index(values) do
+      hidden_input(form, k,
```

Comment:
I am thinking it may be better to pass `id: false` on all of them?

Reply:
Wouldn't that be the only inputs that would be rendered without an `id` then?

Reply:
Yeah, but the odds of customizing them is zero? And this reduce any potential conflict when passing a value as a list. Otherwise the combination of `#{field}_#{value}` may conflict with some `#{field}`?

Reply:
The ID would be `#{field}_#{index}`, but you're right, you probably won't use the ID anywhere.

Reply:
I changed it.

Reply:
Although I must say that I love consistency, so this hurts a bit.

Reply:
Yeah, agreed. Sorry about it, can we please revert?

Reply:
Sure 😄